README.tfdoc.hcl

```diff
@@ -0,0 +1,517 @@
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-module-template"
```

Comment:
`terraform-module-template` should be replaced with  `terraform-google-network-subnet`

seems like you forgot to run `make template/adjust` ? :)